docs/_docs/Starters Academy (LDSSA)/01-Starters-Academy-(Course).md

```diff
@@ -150,4 +178,9 @@ We think that the most important thing you'll take our of the Academy will be th
 
 This resume will list all the Starters Academy's specializations, explain what it means to pass each specialization, and show the grades that you've achieved in each of them. If you miss a specialization, you'll score 0 (zero) on that specialization, and this will be shown in your resume. You'll only be issued a Data Science Resume in case you miss one or zero specializations. 
 
```

Comment:
Changed

Reply:
I would still disambiguate more and replace
"in case you miss one or zero specializations, except for the Bootcamp + Hackathon#1."

by
"in case you miss one or zero specializations, except for the Bootcamp + Hackathon#1, **which cannot be missed**."

I know it might sound redundant. but after all the confusion I've witnessed in the students in Batch#4, I'd say it's worth it. 😅

Reply:
If you read the whole paragraph it says:

>own in your resume. You'll only be issued a Data Science Resume in case you miss one
 or zero specializations, except for the Bootcamp + Hackathon#1.
>
> The capstone, Bootcamp days and Hackathon #1 cannot be missed or delivered late. 


So if we add that it's a bit redundant and you get a bit of weird text:

>own in your resume. You'll only be issued a Data Science Resume in case you miss one
 or zero specializations, except for the Bootcamp + Hackathon#1, which cannot be missed.
>
> The capstone, Bootcamp days and Hackathon #1 cannot be missed or delivered late.